src/config.rs

```diff
@@ -1534,9 +1542,12 @@ impl Config {
 
     #[cfg(windows)]
     pub fn write_defaults() -> io::Result<Cow<'static, Path>> {
-        let path = ::std::env::home_dir()
-            .ok_or_else(|| io::Error::new(io::ErrorKind::NotFound, "could not find profile directory"))
-            .and_then(|mut p| {p.push("alacritty"); p.set_extension("yml"); Ok(p)})?;
+        let mut path = dirs::config_dir()
+            .ok_or_else(|| io::Error::new(io::ErrorKind::NotFound, "could not find profile directory"))?;
+        if !path.exists() {
+            fs::create_dir(&path)?
+        }
```

Comment:
The `create_dir_all` method might make this a little simpler. It doesn't create a directory if it already exists.